### PR TITLE
chore: improves PR template

### PR DESCRIPTION
**Pull request by bartoszmajsak**
## Description

- hides "thank you message" in the comment
- combines two checklists into one
- removed ":" from the headers to make them consistent

## JIRA issue:
N/A

## How Has This Been Tested?
N/A

## Screenshot or short clip:
N/A

## Merge criteria:

- [x] Have a meaningful commit messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
-- [ ] Pull Request contains link to the JIRA issue
-- [ ] Pull Request contains link to any dependent or related Pull Request
+Please complete the following sections for a smooth review.
+-->
 
 ## Description
 <!--- Describe your changes in detail -->
 
-## JIRA issue:
+## JIRA issue
```
Comment (zdtsw) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: i may add "If applicable" for this section as well.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
 
-## Merge criteria:
+## Merge criteria
 <!--- This PR will be merged by any repository approver when it meets all the points in the checklist -->
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 
-- [ ] Have a meaningful commit messages.
+- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
+- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
+- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request
```
Comment (bartoszmajsak) [inline .github/PULL_REQUEST_TEMPLATE.md:27]:
```suggestion
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
 ## How Has This Been Tested?
 <!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- Include details of your testing environment and the tests you ran to -->
 <!--- see how your change affects other areas of the code, etc. -->
 
-## Screenshot or short clip:
-<!--- Attach a screenshot or a short clip demonstrating the feature. -->
+## Screenshot or short clip
+<!--- If applicable,Attach a screenshot or a short clip demonstrating the feature. -->
```
Comment (zdtsw) [inline .github/PULL_REQUEST_TEMPLATE.md:19]:
```suggestion
<!--- If applicable, attach a screenshot or a short clip demonstrating the feature. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (bartoszmajsak) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: Thinking of it... I am not quite sure it deserves a dedicated section. Isn't it enough to add a link in the description, preferably at the end? We actually "ensure" it is added by making it part of the checklist. WDYT?

Comment (zdtsw) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: that would be even better.

**Review by zdtsw — APPROVED** [review]

**Pull request merged**